_overviews/scala3-scaladoc/static-site.md

```diff
@@ -37,6 +37,13 @@ getting-started.html
 
 Scaladoc can transform both files and directories (to organize your documentation into a tree-like structure). By default, directories have a title based on the file name and have empty content. It is possible to provide index pages for each section by creating `index.html` or `index.md` (not both) in the dedicated directory.
 
+Before generating your static site you need to set the `-siteroot` value in your doc `scalacOptions`. The value of this is the directory that holds your docs.  The root URL for the generated documentation will also be `<site-root>`.
+The configuration is as follows:
```

Comment:
Do you think it's okay if I detail the example like this or keep the sentence as you did?
For example if you have a directory called `docs` and you'd like that to be treated as your site root :

```
.
└── docs/
    └── _docs/
        ├── index.html
        └── getting-started.html
```

Then the configuration would be as follows:

Reply:
Actually sure, let's be super explicit and just add this in here as well. It may seem a bit duplicated, but at least then it will make it super clear that `docs` here is what `<site-root>` is up above.